### PR TITLE
Include activation code for RSKIP 219

### PR DESCRIPTION
**Pull request by pamgonzalez**

### CODE REVIEW

File: rskj-core/src/main/java/org/ethereum/config/blockchain/upgrades/ConsensusRule.java [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
 package org.ethereum.config.blockchain.upgrades;
 
-public enum ConsensusRule {
```
Comment (marcos-iov) [inline rskj-core/src/main/java/org/ethereum/config/blockchain/upgrades/ConsensusRule.java:21]: Was this change on purpose?

Comment (guidohernan93) [reply rskj-core/src/main/java/org/ethereum/config/blockchain/upgrades/ConsensusRule.java:21]: I think it was an accidental new line, but should not be like that.

Comment (pamgonzalez) [reply rskj-core/src/main/java/org/ethereum/config/blockchain/upgrades/ConsensusRule.java:21]: Thanks guys, I will resolve it

**Comment by pamgonzalez** [comment]
Use PR: https://github.com/rsksmart/rskj/pull/1494

**Comment by pamgonzalez** [comment]
Use PR: https://github.com/rsksmart/rskj/pull/1494
